### ISSUE TITLE
[PL] [Talent] [Priest] Blessed Resilience

### ISSUE THREAD

**Issue by Vyrtuz** [Atlantiss/BugTracker#7545]
Blessed Resilience (http://wowwiki.wikia.com/wiki/Blessed_Resilience) nie aktywuje się w momencie utraty 10% i więcej punktów zdrowia (tyczy się bezpośrednich jak i obszarowych obrażeń fizycznych i magicznych). Ten talent reaguje prawidłowo tylko w momencie otrzymania bezpośrednich obrażeń krytycznych z czaru/ataku (dla aoe nie sprawdzono).

**Comment by Vyrtuz** [comment]
https://howtopriest.com/viewtopic.php?t=103
"Blessed Resilience is a good choice to take when raiding due to damage mechanics, especially in Heroic encounters. While no direct gain to throughput occurs, the Priest gains an increased chance at survivability through this talent and it will save lives in some cases. A good rule of thumb is that if you are having trouble staying alive during high damage fights due to high damage mechanics, this talent can be a very good investment. Also do not forget that talents like Blessed Resilience can allow the other healers to focus more on other raiders. Majordomo is a fantastic example of where this talent really shines. To get the most out of Blessed Resilience it is recommended that you track the buff and use it to your advantage with Desperate Prayer or Binding Heal."

**Comment by kadi2** [comment]
https://github.com/Atlantiss/BugTracker/issues/6796
Duplicated

**Comment by Vyrtuz** [comment]
I dont think so, of course both spells **only works similarly** but still have diffrient spell's ID's.
